src/GoTrueApi.ts

```diff
@@ -41,6 +41,7 @@ export default class GoTrueApi {
     password: string,
     options: {
       redirectTo?: string
+      hcaptchaToken?: string
```

Comment:
nit: should we just call it `captchaToken` since we are open to exposing other captcha providers in the future?